src/docs/config-strings.dox

```diff
@@ -112,9 +112,9 @@ For example, in Python, code might look as follows:
 
 Because JSON compatibility allows colons to be used as key/value separators,
 WiredTiger URIs may require quoting.  For example, the following
-WT_SESSION::checkpoint call specifies a set of URIs as checkpoint targets,
+WT_SESSION::open_cursor call of a duplicate backup cursor specifies URI log:
```

Comment:
The grammar here is a little off. How about:

```
... the following WT_SESSION::open_cursor call specifies a log URI
using double-quote characters ...
```

Reply:
The only change needed to fix the grammar is to change "specifies URI log:" to "specifies the URI log:".

However, I realized that my explanation is not clear.  Thus I changed the paragraph to:

Because JSON compatibility allows colons to be used as key/value separators,
WiredTiger URIs may require quoting.  For example, the following
WT_SESSION::open_cursor call of a duplicate backup cursor at the end of a
full backup specifies the URI log: within config using double-quote
characters to keep the parser from treating the colon character as a JSON
name/value separator:.

P.S. See a list of backup cursors below from https://source.wiredtiger.com/develop/backup.html
* Full backup: session->open_cursor(session, /* uri */ "backup:", /* to_dup */ NULL, /* config */ NULL, /* cursorp */ &cursor);
* Duplicate or secondary backup cursor at the end of a full backup: session->open_cursor(session, NULL, cursor, "target=(\"log:\")", &dup_cursor);
* Start a log-based incremental backup: session->open_cursor(session, "backup:", NULL, "target=(\"log:\")", &cursor);
* Full backup prior to block-based incremental backup: session->open_cursor(session, "backup:", NULL, "incremental=(enabled=true,this_id=\"ID1\")", &cursor);
* Start a block-based incremental backup: session->open_cursor(session, "backup:", NULL, "incremental=(src_id=\"ID1\",this_id=\"ID2\")", &backup_cur);
* For each file in a block-based incremental backup: A duplicate backup cursor for each file name returned by backup_cursor->next(): session->open_cursor(session, NULL, backup_cur, "incremental=(file=name)", &incr_cur);
* Get existing block incremental identifier strings: session->open_curso(session, "backup:query_id", NULL, NULL, &backup_cur);